extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/deployAzureSQLDBWizard.ts

```diff
@@ -47,14 +53,25 @@ export class DeployAzureSQLDBWizard extends WizardBase<DeployAzureSQLDBWizard, W
 	}
 
 	protected async onOk(): Promise<void> {
-		await this.scriptToNotebook();
+		if (instanceOfAzureSQLDBDeploymentProvider(this.resourceProvider)) {
+			await this.scriptToNotebook();
+		} else {
+			super.onOk();
+		}
+	}
+
+	public refreshWizard() {
+		if (instanceOfAzureSQLDBDeploymentProvider(this.resourceProvider)) {
```

Comment:
Same here - and why do we just do nothing if it isn't that type? Is that going to be expected behavior?

Reply:
This is because SQL DB has multiple different supported types.